### DISCUSSION TITLE
I'm trying to extract a mod

### DISCUSSION THREAD

**Discussion by Santi01saf** [sk-zk/Extractor#17] [Q&A]
Hi, I'm trying to extract a mod but this is what I get, do anyone know how to fix it?

I also found the PMC, PMD, and PMG file location but it seem that it doesn't work, the PMG size is 1KB.
Even sii files too

![image](https://github.com/user-attachments/assets/7c792962-664d-465e-8185-33273fc8c7f5)

![image](https://github.com/user-attachments/assets/046635c7-b292-4e0c-aab0-0a13a4279d1a)

**Comment by sk-zk** [comment]
What is that second screenshot showing? Can you post a link to the mod?

**Reply by Santi01saf** [reply]
It is the cabin.sii

Do you have Discord? So I can send the mod to you there

**Comment by sk-zk** [comment]
Resolved via Discord
